compose/cli/main.py

```diff
@@ -195,6 +208,7 @@ class TopLevelCommand(object):
       --verbose                   Show more output
       --log-level LEVEL           Set log level (DEBUG, INFO, WARNING, ERROR, CRITICAL)
       --no-ansi                   Do not print ANSI control characters
+      --ansi (never|always|auto)  Control when to print ANSI control characters
```

Comment:
should the new option deprecate the old (`--no-ansi`) one? (at least making them conflicting options, as (I think) `--no-ansi` is the equivalent of `--ansi=never`?)

For the `docker` cli, if there's an option that cannot be removed we usually _hide_ the option (to discourage use), and (depending on the case) print a deprecation warning if used (but keep it functional if needed).

Also looking if we should align the UX to the `--progress` option used on `docker build`;

```
      --progress string         Set type of progress output (auto, plain, tty). Use plain to show container output (default "auto")
```

Reply:
Hi @thaJeztah, thanks for your feedback!

`--no-ansi` is indeed the same as `--ansi=never`. In this code, `--no-ansi` overrides `--ansi`, but I could certainly make `--ansi=MODE` and `--no-ansi` mutually exclusive. I could also hide `--no-ansi` and emit a warning.

Alignment across CLIs is good. If `--progress=tty` means "always print ANSI color codes", I dislike that name for it. If that ship has already sailed, and you'd prefer that I follow the auto/plain/tty convention, I can do that.